setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='organizations-edx-platform-extensions',
-    version='1.3.6',
+    version='1.3.7',
```

Comment:
Please do version bump when moving to Integration/Stage and after cherry-picking commits to master.